jest.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     '<rootDir>/development/fitness-functions/**/*.test.(js|ts|tsx)',
   ],
   coverageDirectory: './coverage',
-  coveragePathIgnorePatterns: ['.stories.js', '.snap'],
+  coveragePathIgnorePatterns: ['.stories.*', '.snap'],
```

Comment:
Coverage was failing because it's including `stories.tsx` files

<img width="1440" alt="Screenshot 2023-03-18 at 9 33 18 AM" src="https://user-images.githubusercontent.com/8112138/226120682-b2ef7a4b-bebf-4336-bd97-967206fa7553.png">